provision/vipmanager.bash

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+declare -a ETCD
+
+while getopts "n:e:i:" option; do
+    case "${option}" in
+        n) NODENAME="${OPTARG}" ;;
+	i) VIP_ADDR="${OPTARG}" ;;
+        e) ETCD[${#ETCD[@]}]=$OPTARG ;;
+    esac
+done
+
+
+echo "Installing packages"
+yum install --nogpgcheck --quiet -y -e 0 epel-release
+yum install --nogpgcheck --quiet -y -e 0 jq wget
+VIP_RPM=$(wget -qO- 'https://api.github.com/repos/cybertec-postgresql/vip-manager/releases/latest'| \
+          jq -r '.assets | .[] | select(.content_type == "application/x-rpm") | .browser_download_url')
+wget -q "$VIP_RPM"
+yum install --nogpgcheck --quiet -y -e 0 $(basename "$VIP_RPM")
```

Comment:
Is it possible to use directly `yum install --nogpgcheck --quiet -y -e 0 "$VIP_RPM"` ?

Reply:
good catch I tried it before adding "-r" to "jq" and assumed it couldn't work.
But now it's working.